### PR TITLE
Edited CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by shamilkotta**
# Description
Edited CONTRIBUTING.md file and updated the path in README.md file

## Type of change

Please delete options that are not relevant.

- [x] Non-breaking change which adds infromation

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by harkiratsm** [comment]
image

Can make it aligned

**Comment by shamilkotta** [comment]
@harkiratsm done

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -63,23 +63,62 @@ upstream        https://github.com/commclassroom/commclassroom.git (push)
...
+
+
+Voila :exclamation: You have made a PR to the Community Classroom repository :boom: Sit back patiently and relax while the project maintainers review your PR. Please understand at times the duration can vary from a few hours to a few days.
+
+
+## PR Review
+Your PR will get reviewed soon from the maintainers of the project. If they suggest changes, do all the changes, commit the changes, rebase the branch, squash the commits and push the changes. If everything looks good, your PR will be merged. That's it! Thank you for your contribution! Feel free to suggest any changes to this documentation.
 
+## Note
+Contribution can be very small, that does not matter. We even love to receive a typo fix PR. Adding feature or fixing a bug is not the only way to contribute. You can send us a PR for adding documentation, fixing typos or adding tests.
```
Comment (harkiratsm) [inline CONTRIBUTING.md:123]: mistake  ```tests==texts```

Comment (shamilkotta) [reply CONTRIBUTING.md]: sorry 
fixed now

**Pull request merged**